src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java

```diff
@@ -518,6 +518,9 @@ public void renderShadows(LevelRendererAccessor levelRenderer, Camera playerCame
 		profiler.popPush("shadows");
 		ACTIVE = true;
 
+		RenderBuffers playerBuffers = levelRenderer.getRenderBuffers();
```

Comment:
Could we add a comment that this is needed for mods like Flywheel? Just good to note down why this code is needed so we don't delete it in a refactor.

Reply:
done